libres/lib/include/ert/analysis/update.hpp

```diff
@@ -36,6 +36,20 @@ class update_data_type : public std::enable_shared_from_this<update_data_type> {
         has_observations = true;
     }
 
+    // These functions are needed for pybind to return a writable numpy array
+    // using pybind.
+    Eigen::Ref<Eigen::MatrixXd> get_A() { return A.value(); }
```

Comment:
I would like to discuss this design a bit.
Is it possible to bind the function `load_parameters` directly so that it can be called from Python to generate the matrix `A`?
I think I would prefer not having the function `make_update_data` and the class `update_data_type` and instead just have functions.
We discussed some limitations of Eigen when working with numpy arrays - is this the main reason for the current design?

Reply:
Yes, the update_data_type was always meant to be temporary. We should discuss the design :+1: